### PR TITLE
Describe how to workaround downloads timing out. #143

### PR DESCRIPTION
**Pull request by dabreegster**
Is there github automation that regenerates `README.md`, or do I run something?

### CODE REVIEW

**Comment by agila5** [comment]
Thanks for the PR. 

> Is there github automation that regenerates README.md, or do I run something?

If you are using RStudio you just need to click the "knit" button in the "Source" panel or press "Ctrl + Shift + K", otherwise you need to run `rmarkdown::render("README.Rmd")`. More details [here](https://rmarkdown.rstudio.com/authoring_quick_tour.html).

**Comment by dabreegster** [comment]
I regenerated the readme, but the diff was quite large, including some of the figures. Expected?

File: README.md [hunk 1]
```diff
@@ -114,12 +72,11 @@ Load the package with:
...
 ```
 
-To use alongside functionality in the `sf` package, we also recommend
-attaching this geographic data package as follows:
+To use alongside functionality in the `sf` package, we also recommend attaching this geographic data package as follows:
 
 ``` r
 library(sf)
-#> Linking to GEOS 3.8.0, GDAL 3.0.4, PROJ 6.3.1
+#> Linking to GEOS 3.6.2, GDAL 2.2.3, PROJ 4.9.3
```
Comment (Robinlovelace) [inline README.md:79]: Surprised it works with such old versions of GDAL/PROJ, good to know. FYI @dabreegster there is info here on getting more up-to-date OSGeo packages on Linux (not sure what OS you're running): https://github.com/r-spatial/sf#linux

Massive thank you for the PR btw!

**Comment by agila5** [comment]
> I regenerated the readme, but the diff was quite large, including some of the figures. Expected?

More or less yes, since the images are generated using a `.osm.pbf` file of the Isle of Wight that is updated daily, so the images will change (slight) if you run the README on two different days. See [here](http://download.geofabrik.de/europe/great-britain/england/isle-of-wight.html). The other differences are simply related to the formatting of the `.md` file (i.e. several sentences were collapsed on the same line), probably caused by different versions of `knitr`/`pandoc`. That shouldn't be relevant.  

Thanks! Feel free to open a new issue if you find another bug or if you have any other question/doubt.

File: README.md (hunk 1 above)
Comment (dabreegster) [reply README.md:79]: Ah, thanks for the pointers! I'm on Ubuntu 18.04 and have hit issues with old gdal before, but hadn't found the more up-to-date PPA.

**Pull request merged**